app/src/main/java/com/zionhuang/music/ui/screens/playlist/LocalPlaylistScreen.kt

```diff
@@ -765,35 +765,67 @@ fun LocalPlaylistHeader(
                         )
                     }
 
-                    if (playlist.playlist.browseId != null) {
-                        IconButton(
-                            onClick = {
-                                scope.launch(Dispatchers.IO) {
-                                    val playlistPage = YouTube.playlist(playlist.playlist.browseId).completed().getOrNull() ?: return@launch
-                                    database.transaction {
-                                        clearPlaylist(playlist.id)
-                                        playlistPage.songs
-                                            .map(SongItem::toMediaMetadata)
-                                            .onEach(::insert)
-                                            .mapIndexed { position, song ->
-                                                PlaylistSongMap(
-                                                    songId = song.id,
-                                                    playlistId = playlist.id,
-                                                    position = position
-                                                )
-                                            }
-                                            .forEach(::insert)
+                  // State to control dialog visibility
+                    var showDialog by remember { mutableStateOf(false) }
+
+                    // State for context
+                    val context = LocalContext.current
+                    
+                    // Dialog for confirmation
+                    if (showDialog) {
+        AlertDialog(
+            onDismissRequest = { showDialog = false },
+            title = { Text("Confirm Sync") },
+            text = { Text("Are you sure you want to sync this playlist?") },
```

Comment:
These seem hard-coded? Please put them in the strings.xml file so they can be translated

Reply:
@lucasmz-dev please check now